packages/wallet-manager/src/interfaces/index.ts

```diff
@@ -0,0 +1,57 @@
+import { Web3Provider } from '@ethersproject/providers';
+import { ApiPromise } from '@polkadot/api';
+import { ReactiveController } from 'lit';
+import { AddChain } from '../types';
+import { Signer } from '@ethersproject/abstract-signer';
+
+export interface SupportedWallet {
+  id: string;
+  name: string;
+  icon: string;
+  providerName: string;
+}
+
+export interface IEvmWallet {
+  web3Provider: Web3Provider;
+  address?: string;
+  signer?: Signer;
+  connect(): Promise<void>;
+  addChain({
+    chainId,
+    rpcUrl,
+    chainName
+  }: {
+    chainId: number;
+    rpcUrl: string;
+    chainName: string;
+    nativeCurrency: {
+      name: string;
+      symbol: string;
+      decimals: number;
+    };
```

Comment:
why not use `AddChain` type from `types` file?

Reply:
missed that